repo/packages/H/helloworld/0/resource.json

```diff
@@ -6,35 +6,35 @@
           "contentHash": [
             {
               "algo": "sha256",
-              "value": "f0028105a0bea830d5bdc9f07a3f9c9c3af433a016a6bf1b7921cf646e397376"
+              "value": "7f06eb113794b9e021fb396b65c18c9f2b426a6c2443c0444e359c4be7a6dc89"
             }
           ],
           "kind": "zip",
-          "url": "https://github.com/dcos/dcos-http-cli/releases/download/0.1.0/dcos-http-cli.darwin.zip"
+          "url": "https://gitlab.com/janisz/dcos-http-cli/uploads/36c122df16f0a43015d766ddeb27ba7d/dcos-http-cli.darwin.zip"
```

Comment:
We really shouldn't reference private repos here.

Reply:
I agree but I need a place where I can put this files. We can upload it to S3 but I don't know who to ask for it or create team on gitlab.

Reply:
Looks like it might be the ROQ team: https://wiki.mesosphere.com/display/ENG/ROQ+-+Publishing+to+downloads.mesosphere.com

Reply:
Thanks for link. It's working

downloads.mesosphere.io/dcos-http-cli/0.1.0/dcos-http-cli.darwin.zip
downloads.mesosphere.io/dcos-http-cli/0.1.0/dcos-http-cli.linux.zip
downloads.mesosphere.io/dcos-http-cli/0.1.0/dcos-http-cli.windows.zip

Reply:
Awesome! Thanks for switching it over